README.md

```diff
@@ -60,22 +63,42 @@ Running the Virtual Schema requires a Java Runtime version 11 or later.
 | [Project Keeper Maven Plugin][project-keeper-maven-plugin]         | Checking project structure                             | MIT License                   |
 
 [maven-surefire-plugin]: https://maven.apache.org/surefire/maven-surefire-plugin/
+
```

Comment:
I know that intellij does it automatically now, but please remove the empty lines.

Reply:
opened an issue: https://youtrack.jetbrains.com/issue/IDEA-263256
Until it's fixed I added `<!-- @formatter:off -->` to disable the formatting for that section.